pkg/resources/resources/camel-catalog-3.8.1.yaml

```diff
@@ -1679,12 +1679,6 @@ spec:
     camel-quarkus-knative-producer:
       groupId: org.apache.camel.quarkus
       artifactId: camel-quarkus-knative-producer
-    camel-quarkus-kotlin:
```

Comment:
The catalog is autogenerated and depends on Camel K Runtime. It should not be changed.

Reply:
I removed changes to this file